hs/src/PtrlConsts.hs

```diff
@@ -0,0 +1,14 @@
+module PtrlConsts
```

Comment:
I just noticed the misspelling.  I'll fix this along with anything else that comes up :)

Reply:
fixed